recipes/luple/all/conanfile.py

```diff
@@ -0,0 +1,54 @@
+import os
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.32.0"
+
+class LupleConan(ConanFile):
+    name = "luple"
+    license = "Unlicense"
```

Comment:
https://github.com/alexpolt/luple#license ... It "says" public domain 🤔

Reply:
`Unilicense` is SDPX identifier for public domain: https://spdx.org/licenses/Unlicense.html

Reply:
Thanks! I did not know